public/index.html

```diff
@@ -4,9 +4,8 @@
     <meta charset="utf-8" />
     <link rel="icon" href="Airbnb.ico">
     <title>Vacation Rentals, Homes, Experiences & Places - Airbnb</title>
+    <script src="https://kit.fontawesome.com/bdda782949.js" crossorigin="anonymous"></script>
```

Comment:
if you're use SVG's now, do you still need this font awesome?

Reply:
Good thinking, however, I still do, for the star.